src/components/about/aboutPage.js

```diff
@@ -1,28 +1,17 @@
 "use strict";
 
 var React = require('react');
+var createReactClass = require('create-react-class');

```

Comment:
`React.createClass` was extracted to a separate package in React 16. So updating all React components to import `create-react-class`.